ontologies/ebucore.rdfs.ttl

```diff
@@ -168,10 +168,12 @@ ebucore:synopsis a rdf:Property ;
   rdfs:domain ebucore:Asset .
 
 ebucore:hasCastMember a rdf:Property ;
-  rdfs:label "cast member"@en ;
-  rdfs:label "cast lid"@nl ;
-  rdfs:label "membre de la distribution"@fr ;
-  rdfs:comment "A member of the cast."@en, "Een lid van de cast."@nl, "A member of the cast."@fr, "Range: a string or a Cast"@en ;
+  rdfs:label "cast members"@en ;
+  rdfs:label "castleden"@nl ;
+  rdfs:label "Membres de la distribution"@fr ;
+  rdfs:comment "Members of the cast."@en ;
+  rdfs:comment "Leden van de cast."@nl ;
+  rdfs:comment "Members of the cast."@fr ;
```

Comment:
@lucasotodegraeve Where possible, we try to remain as closely aligned as possible with the originel ontology that is available online, so in this case the EbuCore one. 

We keep the complete ontologies that the data models rely on in the folder https://github.com/viaacode/datamodels/tree/main/external_ontologies_complete and keep the actual used subset in the folder https://github.com/viaacode/datamodels/tree/main/ontologies.

In this case specifically, I would keep the labels singular and just remove the random "Range: a string or a Cast"@en (since this is indeed wrong/strange, although this happens quite a lot with online ontologies).